packages/quantic/cypress/e2e/default-1/case-classification/case-classification-expectations.ts

```diff
@@ -170,6 +170,44 @@ function caseClassificationExpectations(selector: CaseClassificationSelector) {
         })
         .logDetail('should log the "ticket_classification_click" UA event');
     },
+
+    fetchClassificationsAfterValueChange: (
+      field: string,
+      predictions: Array<object>
+    ) => {
+      cy.wait(InterceptAliases.CaseClassification)
+        .then((interception) => {
+          const caseClassificationBody = interception.response?.body;
+
+          expect(caseClassificationBody.fields).to.haveOwnProperty(field);
+          expect(caseClassificationBody.fields[field]).to.haveOwnProperty(
+            'predictions'
+          );
+          expect(
+            caseClassificationBody.fields[field].predictions
+          ).to.deep.equal(predictions);
+        })
+        .logDetail(
+          'should fetch new case classifications after the value changes'
+        );
+    },
+
+    fetchDocumentsAfterValueChange: (mockDocuments: Array<Object>) => {
+      cy.wait(InterceptAliases.DocumentSuggestion)
+        .then((interception) => {
+          const documentSuggestionResponseBody = interception?.response?.body;
+
+          expect(documentSuggestionResponseBody).to.haveOwnProperty(
+            'documents'
+          );
+          expect(documentSuggestionResponseBody.documents).to.deep.equal(
+            mockDocuments
+          );
+        })
+        .logDetail(
+          'should fetch new document suggestions after the value changes'
+        );
+    },
```

Comment:
Same remark, I don't see the point of testing the content of the response, what's the reason behind adding these checks on the response?